Marlin/ultralcd_implementation_hitachi_HD44780.h

```diff
@@ -869,5 +875,5 @@ void lcd_implementation_drawedit(const char* pstr, char* value) {
   }
 
 #endif // LCD_HAS_SLOW_BUTTONS
-
+#endif
 #endif // ULTRALCD_IMPLEMENTATION_HITACHI_HD44780_H
```

Comment:
No. You added on. The addition seems unrelated to your other changes.Ether you fixed a bug, or you introduced one.